### DISCUSSION TITLE
静态库编译报错 Undefined symbols pk::SkStrokeRec....

### DISCUSSION THREAD

**Discussion by kilolumen** [Tencent/libpag#1990] [Q&A] [answered]
<img width="754" alt="截屏2023-12-12 16 56 39" src="https://github.com/Tencent/libpag/assets/7219185/5a702da0-a53e-4675-b806-655b04e1a3e7">

**Comment by domchen** [comment, marked as answer]
使用静态库的前提是要业务自行解决一切符号冲突问题。libpag依赖webp的库，已经内置在里面了，检查一下项目里是不是还有别的地方引用了webp的库，然后跟libpag内置的版本不一致发生了冲突。
